addons/web/static/src/webclient/user_menu/user_menu.xml

```diff
@@ -17,6 +17,7 @@
                     <t t-set-slot="content">
                         <t t-foreach="getElements()" t-as="element" t-key="element_index">
                             <t t-if="!element.hide">
+                                <t t-if="element.type == 'component'" t-component="element.contentComponent"/>
```

Comment:
`===`

Also changes/extra features added in `web` should usually have a test that goes with them. What is the purpose of the feature here?